app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -1987,16 +1992,12 @@ public void changeState(final int state) {
         notifyPlaybackUpdateToListeners();
     }
 
-    private void onPrepared(final boolean playWhenReady) {
-        if (DEBUG) {
-            Log.d(TAG, "onPrepared() called with: playWhenReady = [" + playWhenReady + "]");
```

Comment:
Do not remove this but keep "onPrepared() called"

Reply:
the log is instead to be found at onPlayWhenReadyChanged. ExoPlayer 2.13 splitted up the events in more methods, so I had to rearrange here

Reply:
"Deprecate EventListener.onPlayerStateChanged, replacing it with EventListener.onPlayWhenReadyChanged and EventListener.onPlaybackStateChanged."  

This was the change from the release notes (was actually already in 2.12.0).